gammapy/catalog/fermi.py

```diff
@@ -482,8 +500,11 @@ def lightcurve(self):
         # equally-spaced time intervals. This is roughly correct,
         # for plotting the difference doesn't matter, only for analysis
         time_start = Time("2008-08-04T15:43:36.0000")
-        time_end = Time("2016-08-02T05:44:11.9999")
         n_points = len(flux)
+        if n_points in [8, 48]:
```

Comment:
Is there maybe a clearer way to distinguish this case? If not maybe add a comment what the 8 and 48 means...